### PR TITLE
SDK - Python components - Fixed handling multiline decorators

### PR DESCRIPTION
**Pull request by Ark-kun**
Fixes https://github.com/kubeflow/pipelines/issues/2335

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2345)

### CODE REVIEW

File: sdk/python/tests/components/test_python_op.py [hunk 1]
```diff
@@ -401,8 +401,15 @@ def test_python_component_decorator(self):
         expected_description = 'Sum component description'
         expected_image = 'org/image'
 
-        @python_component(name=expected_name, description=expected_description, base_image=expected_image)
-        def add_two_numbers_decorated(a: float, b: float) -> float:
+        @python_component(
```
Comment (gaoning777) [inline sdk/python/tests/components/test_python_op.py]: I don't see any tests that verify the generated codes of the func that is decorated with python_component?

Comment (Ark-kun) [reply sdk/python/tests/components/test_python_op.py]: You're right. I'll add the execution of this component.

Comment (Ark-kun) [reply sdk/python/tests/components/test_python_op.py]: I've made this test verify the execution.

**Pull request merged**